DeusEx/Classes/DXREnemies.uc

```diff
@@ -139,6 +139,47 @@ function FirstEntry()
 {
     Super.FirstEntry();
     RandoEnemies(dxr.flags.enemiesrandomized);
+    //SwapScriptedPawns();
+}
+
+function SwapScriptedPawns()
+{
+    local ScriptedPawn a, b;
+    local int num, i, slot;
+
+    SetSeed( "SwapScriptedPawns" );
+    num=0;
+    foreach AllActors(class'ScriptedPawn', a )
+    {
+        if( a.bHidden || a.bStatic ) continue;
+        if( a.bImportant ) continue;
```

Comment:
skip bImportant actors because it causes a ton of softlocks, but searching for the bImportant actors you needed for the mission was fun :(